Bump Ionic, fix toggle, use proper validation features

Kukks opened the pull request:
# Conflicts:
#	Chaincase.UI/Chaincase.UI.csproj

DanGould reviewed (commented):
1. On load  NewPasswordPage route: **Unhandled exception** Unable to set property 'ValidationField' on object of type 'Blazor.Ionic.IonInput` ... specified cast is not valid. (same in SSB & iOS)

2. It still does that when I bump to Blazor.Ionic v1.0.9, which this pr could use

<img width="330" alt="Screen Shot 2021-01-04 at 10 32 41 PM" src="https://user-images.githubusercontent.com/8525467/103603517-cf24d000-4edc-11eb-92c3-b0229e4b2c2e.png">

DanGould commented:
I pushed [to my own again](https://github.com/DanGould/chaincase/tree/validation) ... is allow edits by maintainers off? I tried to push here like 3 different ways

Kukks commented:
> I pushed [to my own again](https://github.com/DanGould/chaincase/tree/validation) ... is allow edits by maintainers off? I tried to push here like 3 different ways

Weird, it's on here. Maybe you needed to force push if you rebased?

Kukks commented:
> 1. On load  NewPasswordPage route: **Unhandled exception** Unable to set property 'ValidationField' on object of type 'Blazor.Ionic.IonInput` ... specified cast is not valid. (same in SSB & iOS)
> 2. It still does that when I bump to Blazor.Ionic v1.0.9, which this pr could use
> 
> <img alt="Screen Shot 2021-01-04 at 10 32 41 PM" width="330" src="https://user-images.githubusercontent.com/8525467/103603517-cf24d000-4edc-11eb-92c3-b0229e4b2c2e.png">

Fixed it. It was because we introduced `IonPasswordInput` which did not handle the parameter `ValidationField` correctly and passed it as a string to `IonInput` instead

DanGould commented:
merged via #120
